### DISCUSSION TITLE
Removing package.json

### DISCUSSION THREAD

**Discussion by stephenmathieson** [clibs/clib#232] [Q&A]
Should we remove the `package.json` file from our packages? Since we now support `clib.json`, it seems confusing to see a mix of `package` and `clib` scattered throughout the org.

If this is something we want to do, I'll script it (for my personal repos) and will share the script in case others want to do the same.

**Comment by jwerle** [comment]
Yeah i think we should move away from `package.json` entirely. `clib` will have backwards compat for the file, but will always try to read a `clib.json` file first
